### ISSUE TITLE
creating note from abc notation

### ISSUE THREAD

**Issue by jisike** [saebekassebil/teoria#121]
Are there any functions for creating a note from abc notation?

**Comment by saebekassebil** [comment]
No, I dont think so, but I dont know the ABC notation very well.. I sit possible to give a description of a single note in ABC? If so, i think we could support it.

Otherwise you’d want to write a separate parser, that maybe uses teoria for harmonic purposes

**Comment by jisike** [comment]
Thanks for the reply. Yes you can give a description of a single note with abc notation.

**Comment by jisike** [comment]
Its now possible in tonal with this abc notation extension: http://danigb.github.io/tonal/api/module-abc-notation.html
